modules/RTC/TraceablePeerConnection.js

```diff
@@ -1529,6 +1559,31 @@ TraceablePeerConnection.prototype.setRemoteDescription
 };
 
 /**
+ * Enables/disables video media transmission on this peer connection. When
+ * disabled the SDP video media direction in the local SDP will be adjusted to
+ * 'inactive' which means that no data will be sent nor accepted, but
+ * the connection should be kept alive.
+ * @param {boolean} [active] <tt>true</tt> to enable video media transmission or
+ * <tt>false</tt> to disable. If the value is not a boolean the call will have
+ * no effect.
+ * @return {boolean} <tt>true</tt> if the value has changed and sRD/sLD cycle
+ * needs to be executed in order for the changes to take effect or
+ * <tt>false</tt> if the given value was the same as the previous one.
+ * @public
+ */
+TraceablePeerConnection.prototype.setVideoTransferActive = function(active) {
+    logger.debug(`${this} video transfer active: ${active}`);
```

Comment:
There are couple of ways of doing it. But for example:
```
setMediaTransferActive(type, active) {
    const prop = type === 'audio' ? 'audioTransferActive' : 'audioTransferActive';
    const changed = this[prop] !== active;
    this[prop] = active;
    return changed;
}
```

or you can store the media active state by type initially.

I bet there are other ways too. But I'm OK even with the current implementation.